eng/testing/xunit/vstest.targets

```diff
@@ -1,10 +1,13 @@
 <Project>
   <PropertyGroup>
-    <RunCommand>"$(RunScriptHost)"</RunCommand>
+    <RunCommand>"$(DotNetTool)"</RunCommand>
+    <RunCommand Condition="'$(ContinuousIntegrationBuild)' == 'true'">dotnet</RunCommand>
```

Comment:
Just to understand, could you explain what's the difference in between `DotNetTool` and `dotnet` and why does it has to be different in CI?

Reply:
DotNetTool is a property set by Arcade, mean it's available during the test invocation on your machine. On a different machine you don't want to hardcode the path into the RunTemplate.cmd/sh. There is more work required here anyway to make this work on CI.

My goal in a subsequent PR is to just use dotnet here instead and then for local local development, modify PATH to point to the RunScriptHost if not on CI. As said, I don't want to do this now as this just enables VSTest optionally and not on CI.

Reply:
Another blocker for enabling VSTest in CI is https://github.com/dotnet/runtime/issues/1980.